native/cocos/editor-support/spine-wasm/spine-skeleton-instance.h

```diff
@@ -1,13 +1,11 @@
-#ifndef _SPINE_SKELETON_INSTANCE_H_
-#define _SPINE_SKELETON_INSTANCE_H_
+#pragma once
+
 #include <spine/spine.h>
 
-#include <map>
+#include <cstdint>
 #include <string>
 #include "mesh-type-define.h"
 #include "spine-model.h"
-using namespace spine;
```

Comment:
Don't `using namespace` in header file since it will spread the namespace to all files those include this header.